src/Layout/Box/index.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { space } from 'styled-system';
+
+import { ScreenSize } from '../../Utils/StyleConfig';
+import { SpacingScaleValues } from '../Spacing';
+
+// Responsive
+type BreakpointAliases =
+  | 'default'
+  | 'ms'
+  | 'mm'
+  | 'ml'
+  | 't'
+  | 'ds'
+  | 'dm'
+  | 'dl';
+export const breakpointAliasMap = {
+  ms: `${ScreenSize.mobileS}px`,
```

Comment:
Oh is it because of the library we're using supports the naming like this https://styled-system.com/guides/build-a-box/

Reply:
Oh because we use acronym in spacing properties, I think it is also fine to name breakpoint in the same way for consistency. And it just reminds me that `default` might look not consistent. Will rename it to `_` as the library example below. 

https://styled-system.com/responsive-styles
![image](https://user-images.githubusercontent.com/13249748/96394093-c254e200-11f3-11eb-8b42-1f6c695cbdc4.png)

Reply:
Oh okay, will use the current `ScreenSize`'s keys instead.